libraries/chain/include/eosio/chain/chain_config.hpp

```diff
@@ -115,3 +192,258 @@ FC_REFLECT(eosio::chain::chain_config,
            (max_inline_action_size)(max_inline_action_depth)(max_authority_depth)
 
 )
+//uncomment after adding 1st member to v1 config
+//FC_REFLECT_DERIVED(eosio::chain::chain_config_v1, (eosio::chain::chain_config_v0), CHAIN_CONFIG_V1_MEMBERS())
+
+
+namespace fc {
+
+/**
+ * @brief This is for packing data_entry<chain_config_v0, ...> 
+ * that is used as part of packing data_range<chain_config_v0, ...>
+ * @param s datastream
+ * @param entry contains config reference and particular id
+ * @throws config_parse_error if id is unknown
+ */
+template <typename DataStream>
+inline DataStream &operator<<(DataStream &s, const eosio::chain::data_entry<eosio::chain::chain_config_v0, eosio::chain::config_entry_validator> &entry){
+   using namespace eosio::chain;
+
+   if (!entry.is_allowed())
```

Comment:
Should we consider some sort of logging here to let users know they tried to read something that doesn't exist?

Reply:
this part will be tested after adding first parameter into v1 config. I'll add logs, good idea.